PDXwildlifeDb.sql

```diff
@@ -0,0 +1,283 @@
+﻿DROP TABLE Inf_tbl CASCADE;
+CREATE TABLE  Inf_tbl (
+	InfID	serial PRIMARY KEY,
+	LitterID_fk	int,
+	InfDOB	date,
+	InfName	varchar(25),
+	NumBirth	INTEGER,
+	PandaID_fk	INTEGER
+	--FOREIGN KEY(LitterID_fk) REFERENCES Litter_tbl(LitterID),
+	--FOREIGN KEY(PandaID_fk) REFERENCES Panda_tbl(PandaID),
+);
+-- Copying in from CSV template
+-- The Copy function will write in the PK (serial) variable automatically
+COPY Inf_tbl(InfID,LitterID_fk,NumBirth,InfDOB,PandaID_fk,InfName)
+FROM 'C:\Users\Paul\Desktop\SQL Files\Inf_tbl.csv' DELIMITER ',' CSV HEADER;

```

Comment:
As Aaron mentioned, we can refactor this a bit to work on everyone's machines.  You'll need to add the csv files to the project itself (in some common-sense folder off the root, let's call it X).  Then you can use the 'relative' directory of './' to reference those csv files.  For example, C:\Users\Paul\Desktop\Inf_tbl.csv becomes './X/Inf_tbl.csv'.  As long as you then run the sql from the root of the folder, everything should work.